src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -469,7 +470,7 @@ const ContactDetails = ({
             <AutoSave
               values={values}
               onSave={dispatchSave}
-              debounceDelay={1000}
+              debounceDelay={1000 * 30}
```

Comment:
@sarboc wait is this 30 seconds? Isn't that too long?

Reply:
@christopherhuii The original implementation of this was 1 second after the user became inactive, but it appears to be running every second now. That seems like a lot? I wanted to experiment with tuning this down a bit to see where it gets us.

Reply:
It's hard for me to say. Personally, the jump from 1 second to 30 seconds seems large and I'm not sure what we should be tracking during this experiment. I think 3-5 seconds is a more comfortable jump, but I'm thinking 10 is the absolute max.

Is there any product/design buy in for increasing to 30? If team team agrees on 30 seconds, ignore me.

Reply:
@christopherhuii let me double check with product. The team was ok with 30 seconds at our last eng pairing meeting, but I should def communicate with Helen.